### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by lauramurgatroyd**

### CODE REVIEW

File: .github/workflows/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## Link relevant issues
+
+## Checklist when you are ready to request a review
+
+- [ ] I have performed a self-review of my code
+- [ ] I have added docstrings
+- [ ] CHANGELOG.md has been updated with any functionality change
+- [ ] Request review from all relevant developers
+- [ ] Change pull request label to 'Waiting for review'
```
Comment (paskino) [inline .github/workflows/pull_request_template.md:14]: I think we need the declaration that the contribution is Apache 2.

**Review by paskino — CHANGES_REQUESTED** [review]

File: .github/workflows/pull_request_template.md (hunk 1 above)
Comment (lauramurgatroyd) [reply .github/workflows/pull_request_template.md:14]: we also need to update file headers etc to be Apache2

Comment (lauramurgatroyd) [reply .github/workflows/pull_request_template.md]: updated check list

File: .github/workflows/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Contribution Notes
+
+ - [ ] The content of this Pull Request (the Contribution) is intentionally submitted for inclusion in CIL (the Work) under the terms and conditions of the [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0) License.
```
Comment (paskino) [inline .github/workflows/pull_request_template.md:18]: iDVC rather than CIL

**Review by paskino — CHANGES_REQUESTED** [review]

**Review by paskino — APPROVED** [review]

File: .github/workflows/pull_request_template.md (hunk 2 above)
Comment (lauramurgatroyd) [reply .github/workflows/pull_request_template.md]: Oops! Thanks

**Pull request merged**